css/styles.css

```diff
@@ -0,0 +1,282 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+}
+
+.flex-fix {
+  display: flex;
+  flex-direction: row;
+}
+
+.main-box {
+  margin: auto;
+}
+
+.upper-section {
+  max-width: 100%;
+  height: 40.652em;
+  background: url("../images/mount.jpg") 50% 100% no-repeat;
+  background-size: cover;
+}
+
+.top-box {
+  overflow-y: hidden;
```

Comment:
- [ ] This line `overflow-y: hidden;` is causing this horizontal scrollbar to appear, please remove it 
![image](https://user-images.githubusercontent.com/53256139/88244014-f2720080-cc57-11ea-9fa2-4949147782c6.png)